plot-stdin.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+import os.path
+import sys
+
+# Allow running the script without explicitly setting PYTHONPATH.
+# Not the best solution but hey, as long as it works.
+sys.path += [os.path.dirname(os.path.dirname(os.path.realpath(__file__)))]
+
+import numpy as np
+import typing
+import termios
+import contextlib
+from algutils.utils import *
+from algutils.np_array_to_braille import np_array_to_braille
+from algutils.braille_canvas import BrailleCanvas, Point, Rect
+
+
+def map_coords(coord: Point, src: Rect, dst: Rect) -> Point:
```

Comment:
- Maybe include reversing coordinates in `map_coords` instead of the `list(reversed(` in `__init__`.
- Maybe include reversing coordinate order in `map_coords` instead of adding the local function `__init__.convert`
  - (More generally this is mapping to a mirror image - Where the mirror is `f(x) = y`)
 - Maybe make the function more verbose, because it's hard to understand what `Point(x=dst.left - src.left * scale.x, y=dst.top - src.top * scale.y)` does at this level of verbosity.